### ISSUE TITLE
v3.0.2 dependency issue

### ISSUE THREAD

**Issue by JT501** [0maru/twitter_login#32]
```bash
Because twitter_login >=3.0.2 depends on crypto ^2.0.0 and my_app depends on crypto ^3.0.0, twitter_login >=3.0.2 is forbidden
```

crypto ^2.0.0 does not support `null-safety`

I think it needs to upgrade to crypto ^3.0.0.

**Comment by bojidartonchev** [comment]
Any update on that?

**Comment by 0maru** [comment]
`twitter_login 3.0.1-nullsafety.0` now uses `crypto ^3.0.0-nullsafety.0`.
I think using `twitter_login 3.0.1-nullsafety.0` will solve the problem.

**Comment by 0maru** [comment]
@JT501 @bojidartonchev 
I have released v3.0.4 which resolves the dependencies.
Please use v3.0.4.
